### DISCUSSION TITLE
Joomla 4 Quickstart

### DISCUSSION THREAD

**Discussion by megvadulthangya** [templaza/astroid-framework#282] [General]
Hey! Hello! Is it possible to get a J4 quickstart? I tryed to update the [2.5.4](https://github.com/joomdev/Astroid-Framework/releases/tag/v2.5.4) version, without a succes... Joomla is updated but is not work... Site say: "It seems like the page that you are looking for is no longer here."

**Comment by Chacapamac** [comment]
I @megvadulthangya 
**I will try to help!**

**You said that you try to update the 2.5.4 version to J4?**
— Is that right?
— What template are you using?

• Astroid Version 2.5.4 is the last **Joomdev** Version before they abandon the framework to Github to be retaken by **Templaza**. Updating that version straight to Joomla 4 is not a good idea.

• You should first be sure that Your **Joomdev** version _(framework and Template)_ are really both updated **by using the last Joomdev files** here:
https://github.com/joomdev/Astroid-Framework/releases

I wait for your answer for the followup!

**Reply by megvadulthangya** [reply]
Hello Chacapamac! I used the quickstart what is on the link...

**Comment by Chacapamac** [comment]
Did you update a old website to 2.5.4 or creating a new one?

**Reply by megvadulthangya** [reply]
I tryed a fresh install in WAMP ...

**Comment by Chacapamac** [comment]
Forget **Joomdev** and install from here the last full package version 2.6.3 release the 5th of December 2022
https://github.com/templaza/astroid-framework/releases

**At the top of the page in** —> **Astroid Framework Version 2.6.3**

**Use the Full package** —> astroid-package-v2.6.3.Framework-Template.zip 
![Screen Shot 2023-01-21 at 3 51 36 PM](https://user-images.githubusercontent.com/1907276/213886788-b364b630-4a07-4109-a5ba-757b41f352fc.jpg)

That will install the last version of the **NEW Astroid Framework** and both New **Astroid Zero & One Templates**

**Reply by megvadulthangya** [reply]
Thank you for your help!

**Reply by Chacapamac** [reply]
Happy to Help, Good Day!
